src/Features/Core/Portable/ChangeSignature/DelegateInvokeMethodReferenceFinder.cs

```diff
@@ -106,7 +106,7 @@ protected override bool CanFind(IMethodSymbol symbol)
 
             return invocations.Concat(convertedAnonymousFunctions).SelectAsArray(
                 n => new FinderLocation(n, new ReferenceLocation(document, null, n.GetLocation(), isImplicit: false,
-                    valueUsageInfo: semanticModel.GetValueUsageInfo(n, semanticFactsService, cancellationToken), candidateReason: CandidateReason.None)));
+                    symbolUsageInfo: GetSymbolUsageInfo(n, semanticModel, syntaxFactsService, semanticFactsService, cancellationToken), candidateReason: CandidateReason.None)));
```

Comment:
any reason changed to non-extension method? just asking.

Reply:
GetValueUsageInfo still remains an extension method, but GetSymbolUsageInfo is on AbstractReferenceFinder.